kernel/kernel-impl/src/main/java/org/sakaiproject/email/impl/BasicEmailService.java

```diff
@@ -135,6 +135,9 @@
 	/** Whether to turn on mail debugging */
 	public static final String MAIL_DEBUG = "mail.debug";
 
+	public static final String MAIL_OURDOMAIN = "mail.ourdomain";
```

Comment:
no-reply@sakai.rutgers.edu would be a great FROM address and would allow you Yahoo to accept your mail and would allow you to de-couple your Sakai SPF/DKIM records from your institutional SPF/DKIM records.